jenkins/Dockerfile

```diff
@@ -0,0 +1,76 @@
+# -- DISCLAIMER --
+# This Dockerfile is used by Jenkins to create a testing environment.  It will
+# NOT create a containerized instance of pycds.
+FROM ubuntu:18.04
+
+# Set version for the build stage
+ARG PYTHON_MAJOR=3.6
+ARG PYTHON_PATCH=10
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Set up locales
+RUN apt-get update && \
+	apt-get install -y locales && \
+	sed -i -e 's/# en_US.UTF-8 UTF-8/en_US.UTF-8 UTF-8/' /etc/locale.gen && \
+	dpkg-reconfigure --frontend=noninteractive locales && \
+	update-locale LANG=en_US.UTF-8
+
+ENV LANG en_US.UTF-8
+
+# Set up a repo
+RUN apt-get install -y wget \
+	gnupg && \
+	wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | \
+	apt-key add && \
+	sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt/ bionic-pgdg main" >> /etc/apt/sources.list.d/postgresql.list' && \
+	apt-get -y update
+
+# Install modelmeta requirements
```

Comment:
I think you mean "pycds".

Reply:
Ah, yes I do.  `modelmeta` was the first one I made and the others are all very similar.